digits/model/images/classification/test_views.py

```diff
@@ -825,6 +854,17 @@ class BaseTestCreatedTall(BaseTestCreated):
 class BaseTestCreatedCropInForm(BaseTestCreated):
     CROP_SIZE = 8
 
+class BaseTestCreatedDataAug(BaseTestCreated):
```

Comment:
Indeed! Which ones would you suggest? And the square, and the tall and the wide or just one? Note also it has to be 3 channels to be able to do HSV augmentation.